src/test/clojure/xtdb/sql_test.clj

```diff
@@ -1,7 +1,7 @@
 (ns xtdb.sql-test
   (:require [clojure.java.io :as io]
             [clojure.string :as str]
-            [clojure.test :as t]
+            [clojure.test :as t :refer [deftest]]
```

Comment:
noo, I've only just gotten rid of that! :stuck_out_tongue:

Reply:
Yeah, is there a reason for that? I always never refer but for `clojure.test` it seems idiomatic to me.